tests/compute/test_transform.py

```diff
@@ -2350,6 +2352,15 @@ def test_module_laplacian_pe(idtype):
     else:
         assert F.allclose(new_g.ndata['lappe'].abs(), tgt)
 
+@pytest.mark.parametrize('g', get_cases(['has_scalar_e_feature']))
+@pytest.mark.parametrize('op', ['raw', 'rw', 'gcn', 'ppr'])
+def test_module_sign(g, op):
+    g = g.to(F.ctx())
+    transform = dgl.SIGNDiffusion(k=2, feat_name='h', diffuse_op=op)
+    transform(g)
+    transform = dgl.SIGNDiffusion(k=2, feat_name='h', eweight_name='scalar_w', diffuse_op=op)
+    transform(g)
```

Comment:
Need more thorough unit test. I also suggest testing value correctness using some simple graph and features (such as all one).

Reply:
Updated